src/core/bridge.ts

```diff
@@ -36,6 +45,7 @@ export abstract class AbstractBridge {
 
   constructor(config: BridgeConfig) {
     this.config = Object.assign({}, DEFAULT_CONFIG, config);
+    this.fetch = fetchPonyfill().fetch;
```

Comment:
is there a reason we set this in the constructor rather than with the instance variable declaration?
```js
class AbstractBridge {
  fetch: fetchPonyfill = fetchPonyfill().fetch;
}
```